R/ggbot.R

```diff
@@ -0,0 +1,69 @@
+#' Helper bot to change plot aesthetics
+#'
+#' @md
+#' @param ... character vector containing lexical tokens (a command or a vector or words)
+#'
+#' @details Commands currently supported:
+#' - change colour of points
+#' - remove axis, legend, everything
+#'
+#' @return a \code{\link[ggplot2]{theme}} object which can be used in
+#'   \code{\link[ggplot2]{ggplot2}} calls.
+#' @export
+#'
+#' @examples
+#' library(ggplot2)
+#' ggplot(mtcars, aes(cyl, hp, col = "red")) +
+#'   geom_point() +
+#'   ggbot("blue points")
+ggbot <- function(...) {
+
+    tokens <- unlist(strsplit(unlist(rlang::exprs(...)), " "))
```

Comment:
With multiple arguments, this mashes all the words together into one vector.  Perhaps distinct string inputs should be considered separate commands? e.g. `ggbot("rotate x axis", "remove y axis")` is something that could supported, and the words in each of those commands would ideally be kept separate.  Unless the idea is that there should only one command per ggbot call -- in which case combining all the words together makes sense.

Reply:
The use case I had in mind was either one whole command or a vector of words. The use case you suggest is probably much more likely though; I should iterate over the elements of `...` and treat each one as a call to `ggbot`.